lib/loader/index.d.ts

```diff
@@ -100,22 +100,25 @@ export interface ASUtil {
 }
 
 /** Asynchronously instantiates an AssemblyScript module from anything that can be instantiated. */
-export declare function instantiate<T extends {}>(
+export declare function instantiate<T extends Record<string,unknown>>(
```

Comment:
I believe the correct type should be `[key: string]: Record<string, any>` and not `Record<string, unknown>`.

Is there a reason why we can't use `any` here?

Reply:
What I'm mostly interested in is whether it breaks your code the way it is now or not. From your comment, I assume it does?

Reply:
Steps I used to test this:

1. Copied the file to a local installation of AS
2. imported the loader
3. ran tsc and asserted there were no compile time errors using latest stable typescript

My software did not break.

I also suppose the answer to my question is that you prefer not to use `any` if possible. This is fair.  Thanks for this work! It's going to help a lot of people in the long run.

Reply:
The problem with `any` is mostly that the linter will complain, and we'd have to add an `// eslint-disable` directive where used, which I'm trying to avoid :)